### PR TITLE
Upgrade to cover the functionality used in Terraform Provider v2.6.0

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #538

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## 0.18.2 (Unreleased)
 
+- Update to v2.6.0 of the AWS Terraform Provider.
```
Comment (lukehoban) [inline CHANGELOG.md]: Minor - Given that it looks like we didn't release after the update to v2.5.0, we may be able to just leave off that line of the CHANGELOG?

Comment (stack72) [reply CHANGELOG.md]: sure :) I will do a similar thing in the azure provider PR as well

Comment (stack72) [reply CHANGELOG.md]: done!

**Comment by jen20** [comment]
Fixes #538.

**Pull request merged**